tests/unit/steps/test_steps.py

```diff
@@ -140,11 +140,11 @@ def test_step_str():
 def test_step_str_dependent():
     """Test BaseStep string representation."""
     expected = (
-        f"a\n\ta.a\n\t\t{DEPENDENCY_DESCRIPTION_PREFIX}a.a.a\n"
-        f"\t\t{DEPENDENCY_DESCRIPTION_PREFIX}a.a.b\n\ta.b\n"
+        f"a\n\ta.a\n\t\tΨ {DEPENDENCY_DESCRIPTION_PREFIX}a.a.a\n"
```

Comment:
The test is saying `test_step_str_dependent`, can we do this test in another test or renamed it.

Reply:
We already have `test_plan_structure` doing this. I'll update the test docstring there and I think we are covered.